src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -497,6 +497,10 @@ def update_rem_if_mgmt(self, if_oid, if_name):
             return
         try:
             mgmt_ip_str = lldp_kvs[b'lldp_rem_man_addr'].decode()
+            mgmt_ip_str = mgmt_ip_str.strip()
+            if len(mgmt_ip_str) == 0:
+                # the peer advertise an emtpy mgmt address
+                return
```

Comment:
why is this related to lldp?

Reply:
The peer side also has the same use case, ie. no mgmt ports configured and connected. As comment "the peer advertise an emtpy mgmt address"